source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ACVTableViewController.m

```diff
@@ -7,9 +7,8 @@
 
 #import "ACVTableViewController.h"
 
-@implementation ACVTableViewController
-{    
-    NSArray<NSString*> *pathsToFiles;
+@implementation ACVTableViewController {
```

Comment:
this feels a little weird -- here (and in a few other spots) we're moving a brace up to EOL, but below in some places we're  leaving them below (see later in this file) or explicitly moving them from EOL to the next line (see `ADCResolver.m`)

Reply:
Yes, it does feel weird, and I had to crosscheck few times to be sure.

this one is a @implementation section, not function definition, that's the reason they looks different from each other.


---
In reply to: [300182338](https://github.com/microsoft/AdaptiveCards/pull/3146#discussion_r300182338) [](ancestors = 300182338)